lib/src/migrators/module/references.dart

```diff
@@ -90,9 +90,9 @@ class References {
   /// map to the [ReferenceSource] for the `sass:meta` module).
   final Map<SassNode, ReferenceSource> sources;
 
-  /// The set of import-only files that do not directly depend on their regular
-  /// counterparts.
-  final Set<Uri> orphanImportOnlyFiles;
+  /// Map of import-only files that do not directly depend on their regular
+  /// counterparts to the last forward appearing within it.
```

Comment:
Mention that the value may be `null` if the file contains no `@forward`s.

Reply:
Done.